src/main/java/com/zextras/utils/ObjectFactory.java

```diff
@@ -1,12 +1,20 @@
 package com.zextras.utils;
 
+import com.zextras.persistence.SlcwException;
 import java.lang.reflect.InvocationTargetException;
 
 /**
  * Helper util class that creates new objects.
  */
 public class ObjectFactory {
 
+  /**
+   * Private constructor used to restrict someone from instantiating utility class.
+   */
+  private ObjectFactory() {
+    throw new SlcwException("Utility class can not been instantiated.");
```

Comment:
Please replace the following with the proposed everywhere
>"Utility class can not been instantiated."

"Utility class cannot be instantiated"